export const testHeroPage = async () => {
  try {
    console.log("test");
  } catch (error: any) {}
};
